backend/composer/services/cs_ingestion/cs_ingestion_services.py

```diff
@@ -14,10 +14,10 @@
 logger_service = LoggerService()
 
 
-def ingest_statements(update_upstream=False, update_anatomical_entities=False, should_overwrite=True, full_imports=[], label_imports=[]):
+def ingest_statements(update_upstream=False, update_anatomical_entities=False, disable_overwrite=False, full_imports=[], label_imports=[]):
```

Comment:
issue: Same as in the other PR, maybe it makes more sense to change here given the target of the PR